src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Link } from 'gatsby';
+
+import netlify from './netlify.svg';
+import './Footer.css';
+
+const Footer = () => (
+  <footer className="footer">
+    <div className="footer__wrapper">
+      <p className="footer__phrase">
+          Made with ❤ by
+        {' '}
```

Comment:
Why do we have to interpolate spaces?

Reply:
In order to avoid having 180+ character lines & to avoid using `&nbsp;`.
Although it's not mentioned in the [official](https://github.com/airbnb/javascript) documentation. My automatic formatter did this, while following the airbnb configuration.

Would you prefer another way?